docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
+Instructions for the latest code from the `main` branch. Note the 
+installation differs between the two available OSs. On Jetson Nano you will 
+need to install Jetpack 4.6.2. If you have newer Jetson board or a Jetson 
+Xavier you need to install Jetpack 5.0.2.
+
+
+### Installation on Jetson Nano
+
+
+* [Step 1b: Flash Operating System](#step-1b-flash-operating-system)
+* [Step 2b: Free up the serial port](#step-2b-free-up-the-serial-port-optional-only-needed-if-youre-using-the-robohat-mm1)
+* [Step 3b: Setup Python Environment](#step-3b-setup-python-environment)
+* [Step 4b: (Optional) Install PyGame for USB camera](#step-4b-optional-install-pygame-for-usb-camera)
+
+
+#### Step 1b: Flash Operating System
+
+These instructions work for Jetpack 4.6.1. 
+
+* If you have a 4gb Jetson Nano then download Jetpack 4.6.1 from Nvidia
+  here: [jetson-nano-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jeston_nano/jetson-nano-jp461-sd-card-image.zip)
+* If you have a 2gb Jetson Nano the download Jetpack 4.5.1from Nvidia
+  here: [jetson-nano-2gb-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jetson_nano_2gb/jetson-nano-2gb-jp461-sd-card-image.zip)
+
+
+Visit the official [Nvidia Jetson Nano Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit#prepare)
+or [Nvidia Xavier NX Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit).
+Work through the __Prepare for Setup__, __Writing Image to the microSD Card__,
+and __Setup and First Boot__ instructions, then return here.
+
+Once you're done with the setup, ssh into your vehicle. Use the terminal for
+Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+for windows.
+
+
+Remove Libre Office:
 
 ```bash
-cd ~/projects
+sudo apt-get remove --purge libreoffice*
+sudo apt-get clean
+sudo apt-get autoremove
+```
+
+And add a 6GB swap file:
+
+```bash
+git clone https://github.com/JetsonHacksNano/installSwapfile
+cd installSwapfile
+./installSwapfile.sh
+reboot 
+```
+
+#### Step 2b: Free up the serial port (optional. Only needed if you're using the Robohat MM1)
+
+```bash
+sudo usermod -aG dialout <your username>
+sudo systemctl disable nvgetty
+```
+
+
+#### Step 3b: Setup Python Environment
+
+The installation on JP 4.6.X is more tricky, because we need to use a newer
+version of tensorflow than what is packaged by NVidia for this OS version.
+Also, we need a compatible OpenCV version which needs to be built on the nano.
+
+* Step 1: Install mamba-forge
+
+Download and install mambaforge
+
+```bash
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh
+chmod u+x ./Mambaforge-Linux-aarch64.sh
+bash ./Mambaforge-Linux-aarch64.sh
```

Comment:
I was unsure at this step but went with `yes`:
```
Do you wish the installer to initialize Mambaforge
by running conda init? [yes|no]
[no] >>> yes
```

Reply:
Ok, makes sense, updated the doc.